sessions/biases-in-delay-distributions.qmd

```diff
@@ -217,12 +230,22 @@ Can you establish under which conditions the bias in estimation gets worse?
 # Real-time estimation: truncation
 
 The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
-However, information on delay distribution is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
-
-Estimating delays in real time comes with particular challenges, as the timing of the cut-off might introduce a bias.
-If, for example, infections are exponentially increasing then there will be disproportionately more people with recent symptom onset.
-Without adjustment, this would artificially decrease the estimate of the mean delay compared to its true value for all infections.
+However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+
+Estimating delays in real time comes with particular challenges, as we know that our data are usually incomplete: not enough time has passed to observe all of the relevant events that could have happened.
+We call this incompleteness *truncation*.
```

Comment:
```suggestion
We call this incompleteness *right truncation*.
```

Reply:
I think we probably want to be specific here as with censoring so people aren't confused we could also give the general definition but I don't think we need to.